lockbox-ios/View/ItemListView.swift

```diff
@@ -65,6 +65,7 @@ extension ItemListView: ItemListViewProtocol {
     func bind(scrollAction: Driver<ScrollAction>) {
         scrollAction.delay(RxTimeInterval(0.1))
                     .drive(onNext: { action in
+                        guard self.tableView.dataSource?.tableView(self.tableView, numberOfRowsInSection: 0) ?? 0 > 0 else { return }
```

Comment:
This fixes a launch bug as experienced at #923, it was getting annoying for my testing...